kerastuner/abstractions/display.py

```diff
@@ -28,15 +28,17 @@
 
 
 # Check if we are in a ipython/colab environement
-try:
-    class_name = get_ipython().__class__.__name__
-    if "Terminal" in class_name:
-        IS_NOTEBOOK = False
-    else:
-        IS_NOTEBOOK = True
-
-except NameError:
-    IS_NOTEBOOK = False
+def is_notebook():
+    try:
+        class_name = get_ipython().__class__.__name__
+        if "Terminal" in class_name:
+            return False
+        else:
+            return True
+
+    except NameError:
+        return False
+IS_NOTEBOOK = is_notebook()
```

Comment:
Should add two blank lines before `IS_NOTEBOOK` (2 lines b/t top-level definitions is standard)